src/styles/components/_home-search-box.scss

```diff
@@ -1,15 +1,73 @@
 @import '../abstracts/_abstracts-dir';
 
 .home-search-box {
-  @include flex-center();
+  width: 80%;
+
+  background-color: $light;
+  box-shadow: rem(5px 10px 20px 0px) hsla(0, 0%, 0%, 0.16);
+
+  @include respond-above('md') {
+    width: 90%;
+  }
+
+  & > input {
+    width: 75%;
+    padding: rem(5px 10px 5px 10px);
+
+    color: $dove-gray;
+
+    border: none;
+    font-size: rem(14px);
+    line-height: 1.23;
+    text-align: left;
+
+    @include respond-above('sm') {
+      padding: rem(10px 20px 10px 20px);
+      font-size: rem(16px);
+    }
+
+    @include respond-above('md') {
+      padding: rem(20px 40px 20px 40px);
+      font-size: rem(20px);
+    }
+
+    @include respond-above('lg') {
+      padding: rem(35px 60px 35px 60px);
+      font-size: rem(24px);
+    }
+  }
```

Comment:
Since the `input` is an element in `home-search-box`. You could do this though 🤔
```suggestion
  &__input {
    width: 100%;
    box-sizing: border-box;
    padding: rem(5px 10px 5px 10px);

    color: $dove-gray;

    border: none;
    font-size: rem(14px);
    line-height: 1.23;

    @include respond-above('sm') {
      padding: rem(10px 20px 10px 20px);
      font-size: rem(16px);
    }

    @include respond-above('md') {
      padding: rem(20px 40px 20px 40px);
      font-size: rem(20px);
    }

    @include respond-above('lg') {
      padding: rem(35px 60px 35px 60px);
      font-size: rem(24px);
    }
  }
```

Reply:
First, I try to make the input field appear `75%` so it's not overlapping with the button. but I'll try yours. Thx.

Reply:
Avoid using `>` (css selector) when it's already an element.

Reply:
Updated: I decided to use input element with `type="search"` instead of `type="text"`
And we've got a new element to discard what we've searched like this:
<img width="476" alt="Screenshot 2564-02-28 at 21 58 00" src="https://user-images.githubusercontent.com/32285705/109423038-9d2d7700-7a10-11eb-92c3-dc0e773dcb5c.png">

Obviously, see that overlaps with `search-icon` so I decided to use `width:75%` as previous. And it went like this: 
<img width="463" alt="Screenshot 2564-02-28 at 21 59 09" src="https://user-images.githubusercontent.com/32285705/109423087-c221ea00-7a10-11eb-84f3-5d48ae3a9105.png">

**ignore the blue overlay or the cross icon first**
What do you think? 
1. Do we need this type? or just stick with the same
2. About the change of width of the input field.

Reply:
Nice to have `search`!